host-configs/LLNL/quartz-base.cmake

```diff
@@ -58,5 +58,12 @@ set(MKL_LIBRARIES ${MKL_ROOT}/lib/intel64/libmkl_intel_lp64.so
 
 # ATS
 set(ATS_ARGUMENTS "--machine slurm36"  CACHE STRING "")
+set(USER $ENV{USER} CACHE STRING "")
+set(ATS_WORKING_DIR "/p/lustre2/${USER}/integratedTests/${CONFIG_NAME}"  CACHE PATH "")
+set(ATS_BASELINE_DIR "/p/lustre2/${USER}/integratedTests/baselines"  CACHE PATH "")
```

Comment:
These are setting where we want to run the tests and where we want to look for baselines.  Currently, there are no default values set, which triggers a cmake error later on.

Reply:
This seems like a reasonable default for LC systems. Otherwise we could simply define a the geos directory as a default.